customize.sh

```diff
@@ -126,7 +126,7 @@ grep -q '\.' "$TMPDIR/SYN_CONF.txt"; if [ $? -eq 0 ]; then
 	ui_print ""
 	echo -e "!- Your "$CONF" file contain error(s)"
 	ui_print "Now copying new Detach file, check and try again."
-	cp -afr "$TMPDIR/Detach.txt" "$DetachFile"
+	cp -fr "$TMPDIR/Detach.txt" "$DetachFile"
```

Comment:
Same here

Reply:
It's necessary in this case. As it's now necessary to overwrite already existing file.